src/applications/personalization/profile/components/ProfileInformationEditView.jsx

```diff
@@ -188,13 +188,14 @@ export class ProfileInformationEditView extends Component {
 
     // for personal info fields we are using a different request flow
     if (Object.values(PERSONAL_INFO_FIELD_NAMES).includes(fieldName)) {
-      this.props.createPersonalInfoUpdate(
-        apiRoute,
-        'PUT',
+      this.props.createPersonalInfoUpdate({
```

Comment:
Changed the argument on this function to be an Object bc there were getting to be way too many arguments otherwise.